_includes/header.html

```diff
@@ -1,5 +1,28 @@
 <header class="site-header">
+<script type="text/javascript">
+    window.onload = function(e) {
```

Comment:
I think this logic can be refactored to a one liner ternary eg: 

` window.document.getElementsByTagName('html')[0].className = window.document.getElementById('toggleTheme').checked ? "dark":"default"; `
maybe this could be called applyTheme() and be called when the window is being loaded, and when the button is pressed? 

Else it really looks neat, I need to clone it and test the theme, but thanks a lot for your effort so far, we're almost there ! :)